comms/src/protocol/messaging/protocol.rs

```diff
@@ -61,6 +61,7 @@ const MAX_SEND_RETRIES: usize = 1;
 /// The maximum amount of inbound messages to accept within the `RATE_LIMIT_RESTOCK_INTERVAL` window
 const RATE_LIMIT_CAPACITY: usize = 10;
 const RATE_LIMIT_RESTOCK_INTERVAL: Duration = Duration::from_millis(100);
+const MAX_FRAME_LENGTH_MB: usize = 8;
```

Comment:
nit: We may want to do sub-Mb adjustment 
```suggestion
const MAX_FRAME_LENGTH: usize = 8 * 1_024 * 1_024;
```

Reply:
Committed, rebased and force pushed from local